stacks-signer/CHANGELOG.md

```diff
@@ -11,10 +11,25 @@ and this project adheres to the versioning scheme outlined in the [README.md](RE
 
 ### Changed
 
+## [3.0.0.0.2.0]
+
+### Added
+
```

Comment:
Maybe we should replicate the comment from the other changelog here as well:

- If the winning miner of a sortition is committed to the wrong parent tenure, the previous miner can immediately tenure extend and continue mining since the winning miner would never be able to propose a valid block. (#5361)

Reply:
ec48c5833a49d65540e8d9c943db260f3f9d3f19